### ISSUE TITLE
The "Note" field should be cleared whenever user click on the "add more" (check with plus symbol) button

### ISSUE THREAD

**Issue by hailduong** [mtotschnig/MyExpenses#404]
Whenever i click on the "Add more" (the check with a plus sign), i would expect that the note be cleared out, since I always have a different note for what i am buying etc... for each item.

**Comment by klawdhfzasjhaa** [comment]
This would be possible if #398 was implemented.

**Comment by mtotschnig** [comment]
@klawdhfzasjhaa Not exactly, since #398 refers to the auto-fill based on Payer/Payee, and this ticket refers to the "Save and new" button". Thus probably a separate configuration option would be needed here.
